shared/libraries/native_streaming_protocol/src/base_session_handler.cpp

```diff
@@ -35,11 +35,11 @@ void BaseSessionHandler::startConnectionActivityMonitoring(Int heartbeatPeriod,
         connectionInactivityTimer->cancel();
         connectionInactivityTimer->expires_from_now(inactivityTimeoutMs);
         connectionInactivityTimer->async_wait(
-            [this](const boost::system::error_code& ec)
+            [errorHandler = errorHandler, session = session](const boost::system::error_code& ec)
             {
                 if (ec)
```

Comment:
What does this mean? If operation failed we return and do nothing?

Reply:
We do smth (here triggering connection closing) when timer expires. If timer callback called with some error code (for example timer operation cancelled on shutdown)  - yes, we return do nothing.

Reply:
Ok